docs/onlinebooking.html

```diff
@@ -24,6 +24,7 @@ <h1>Online Booking</h1>
         element: document.getElementById('onlinebooking'), // Required
         locale: 'nl_NL', // Optional, default is nl_NL. Valid values are de_DE, en_GB, and nl_NL
         //package_id: 7, // Optional, this will skip the package selection step for online booking
+        package_ids: [7, 26, 29], // Optional, this will only show these packages instead of all of them. Option "package_id" has priority over this option.
```

Comment:
I think having `package_id` take precedence over `package_ids` makes little sense. It seems to me that `package_ids = [2]` should be equivalent to `package_id = 2`